src/utils/handlers/handleFormSubmit.ts

```diff
@@ -6,7 +6,7 @@ async function handleFormSubmit<T>(
   setChanged: (changed: boolean) => void
 ): Promise<boolean> {
   const errors = await onSubmit(data);
-  const ok = errors.length === 0;
+  const ok = errors?.length === 0;
```

Comment:
Why?

Reply:
Code fails here occasionally looking at Sentry but the root cause of this issue seems to be returning non-array type from onSubmit